learners/checkout.md

```diff
@@ -17,7 +17,7 @@ Briefly, the three steps are:
 
 #### How will the Checkout steps be tracked?
  **For the “Get Involved” step, trainees must login to AMY to report which participation option they would like to have counted.**
- Welcome Sessions and Teaching Demonstrations will be tracked automatically based on data shared by session hosts.
+ Welcome Sessions and Teaching Demonstrations will be tracked automatically based on data shared by session hosts. You can track your progress at https://amy.carpentries.org/.
```

Comment:
```suggestion
 Welcome Sessions and Teaching Demonstrations will be tracked automatically based on data shared by session hosts. You can track your progress in [AMY](https://amy.carpentries.org/).
```

plain URLs are bad for screen readers and this is not in an exercise, so making it better